src/tests/places/controllers/get-places.e2e-spec.ts

```diff
@@ -76,6 +78,8 @@ describe('PlacesController (e2e)', () => {
           nameSlug: 'zhp-test',
           demands: [],
           priority: 0,
+          state: 1,
+          transitions: [{ startState: PlaceState.ACTIVE, endState: PlaceState.INACTIVE, name: 'DEACTIVATE' }],
```

Comment:
Could you use an enum for the name of transition? Then you can reuse the enum in DTO